Colornames

ntwcklng opened the pull request:
I had to move the simple colornames to the bottom in order to get the tests passed.

ntwcklng committed:
added new colors

ntwcklng committed:
fixed failed test

timche commented on index.js:
```diff
@@ -102,11 +102,118 @@ var mapValues = {
+    'darkorchid': 'dunkleorchidee',
+    'darkred': 'dunkelrot',
+    'darksalmon': 'dunklelachsfarbe',
+    'darkseagreen': 'dunkleseegrün',
```
Typo -> dunkle**s**seegrün

timche commented on index.js:
```diff
@@ -102,11 +102,118 @@ var mapValues = {
+    'darkred': 'dunkelrot',
+    'darksalmon': 'dunklelachsfarbe',
+    'darkseagreen': 'dunkleseegrün',
+    'darkslateblue': 'dunkleschieferblau',
```
Typo -> dunkle**s**schieferblau

ntwcklng committed:
typos

ntwcklng committed:
typos

timche commented:
Nice PR 😄 Can you tell me where you found the translations or have you done it by yourself? I just want to make sure the translations are correct, because I'm not a expert in (german) colors 😁

ntwcklng commented:
Thanks :) You can find them here: http://www.sttmedia.com/colornames
I removed the unimportant ones and changed some names so they make more sense.
With the sublime package color-highlighter you can check if all the original colornames are valid
![screenshot at aug 26 12-51-59](https://cloud.githubusercontent.com/assets/8714775/9491700/47f96b84-4bf1-11e5-9352-7fcf2ae85a58.png)

timche commented:
Alright, thanks :) I will merge it.

timche commented:
What's your Twitter handle? I would like to credit you.

ntwcklng commented:
Thanks, my Twitter handle: @ntwcklng
